src/client.toit

```diff
@@ -205,7 +205,14 @@ class Client:
     // is critical that the packet bits sent over the transport stream
     // aren't interleaved, so we use a mutex to serialize the sends.
     sending_.do:
-      transport_.send packet
+      exception := catch:
```

Comment:
Maybe you should be using `--trace=(: should_trace_exception it)` here?

Reply:
done.